### PR TITLE
git: Add ImGui as submodule

### PR DESCRIPTION
**Pull request by lafette21**
- Modify LICENSE to satisfy the chain

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
-## Bibliography
+[![Build](https://github.com/lafette21/yui-cpp/actions/workflows/build.yml/badge.svg)](https://github.com/lafette21/yui-cpp/actions/workflows/build.yml)
 
-[CMake Conan Boilerplate](https://github.com/cpp-best-practices/cmake_conan_boilerplate_template)
+## Project structure
 
-[Project Options](https://github.com/aminya/project_options)
+```
+.
+├── 3pp
```
Comment (GK72) [inline README.md]: `3pp` or `deps`? I personally use `deps`, many projects use that.

**Review by GK72 — COMMENTED** [review]
Maybe example codes to illustrate the usage of the dependency.

File: README.md (hunk 1 above)
Comment (lafette21) [reply README.md]: I like 3pp more to be honest. gRPC uses `third_party` (https://github.com/grpc/grpc/tree/master/third_party). spdlog includes it into the `include` folder (https://github.com/gabime/spdlog/tree/v1.x/include/spdlog/fmt).

Comment (lafette21) [reply README.md]: However there is a possibility to use 2pp dependencies as well so I'm not sure.

**Review by GK72 — APPROVED** [review]

**Pull request merged**